public/index.html

```diff
@@ -2,12 +2,17 @@
 <html>
   <head>
     <title>Airbnb Experiences</title>
+    <meta charset="utf-8">
```

Comment:
Good to know that meta tags need to go in proxy html, I was wondering about this earlier